content-src/components/DiscoveryStreamComponents/List/List.jsx

```diff
@@ -68,7 +68,7 @@ export function _List(props) {
 
   return (
     <div>
-      <h3 className="ds-list-title">{props.header && props.header.title}</h3>
+      {props.header && props.header.title ? <div className="ds-header">{props.header.title}</div> : null }
```

Comment:
If we were to make this a component (new bug is fine with me), I can see value in being able to do validation for internal required props so it doesn't choke in one place.

Reply:
Yeah it could be a very tiny component if you'd like to add that to this PR. There is some value in that for sure.

Reply:
I would rather keep focus on spocs if possible, so would rather punt this.